projects/moremoney/index.js

```diff
@@ -0,0 +1,244 @@
+/*==================================================
+  Modules
+  ==================================================*/
+
+const sdk = require("@defillama/sdk");
+
+const IsolatedLending = "0xDc5CCAAA928De5D318605A76eEDE50f205Aa6D93";
+const CurvePoolRewards = "0x9727D535165e19590013bdDea8Fd85dd618b9aF7";
+const account = "0x0000000000000000000000000000000000000000";
+const tokenList = require("./tokenlist.json");
```

Comment:
can we get the tokenlist in a way that auto-updates (eg: by querying smart contracts)? seems like this is likely to get stale with time

Reply:
@0xngmi can you have a look again, just updated the code to query the contracts instead of searching for the info locally on the `tokenlist.json` file

Reply:
Added back the `TokenList.json` so if the `decimals` is not available from the smartcontract, we have the `tokenlist.json` as a backup to lookup.